example/src/stories/HighlightText.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {HighLightText} from 'rn-base-component'
+
+export default {
+  title: 'components/HighlightText',
+  component: HighLightText,
+} as ComponentMeta<typeof HighLightText>
+
+export const Basic: ComponentStory<typeof HighLightText> = args => <HighLightText {...args} />
+
+Basic.args = {
+  textToHighlight: 'Hello STS Tea123123m!',
```

Comment:
Doest this text "Tea123123m" have meaning?

Reply:
No meaning. Just for testing the text to highlight